### ISSUE TITLE
 Starting new HTTP connection

### ISSUE THREAD

**Issue by carlosvega** [EpistasisLab/tpot#999]
Every time I execute TPOT I see these lines one the terminal:

> 	|DEBUG @ _new_conn ::: Starting new HTTP connection (1): updatechecker.bryceboe.com:80
> 	|DEBUG @ _make_request ::: http://updatechecker.bryceboe.com:80 "PUT /check HTTP/1.1" 200 23

Why is making a connection, why is this enabled by default, and how can I disable it?

**Comment by carlosvega** [comment]
okey so

> disable_update_check: boolean, optional (default=False)
> Flag indicating whether the TPOT version checker should be disabled.
> 
> The update checker will tell you when a new version of TPOT has been released.

**Comment by weixuanfu** [comment]
Yes, it is the update checker and you can disable it via disable_update_check=True.
